README.md

```diff
@@ -171,7 +171,11 @@ This will provide the following tasks:
 - `extension:test_app`, which generates a dummy app for your extension
 - `extension:specs` (default), which runs the specs for your extension
 
-It also allows you to run `rake` to, respectively, generate a test app and run all tests.
+If your extension requires the `test_app` to be always recreated you can do so by running:
+
+```rb
+bin/rake extension:test_app extension:specs
```

Comment:
Remember that there is no `bin/rake` in extensions generated with solidus_dev_support.

Reply:
This always gets me, I was sure I saw it in the README 🤦‍♂️